app/templates/output_document.html.erb

```diff
@@ -162,7 +162,16 @@
 
     <p><b>Total value of your pension pot:</b></p>
 
-    <p class="data"><%= @appointment_summary.value_of_pension_pots %></p>
+    <p class="data">
+      <% if @appointment_summary.value_of_pension_pots.blank? %>
+        Unknown
+      <% elsif @appointment_summary.upper_value_of_pension_pots.blank? %>
+        <%= number_to_currency @appointment_summary.value_of_pension_pots, precision: 0 %>
+      <% else %>
+        <%= number_to_currency(@appointment_summary.value_of_pension_pots, precision: 0) + ' - ' +
+                number_to_currency(@appointment_summary.upper_value_of_pension_pots, precision: 0) %>
+      <% end %>
```

Comment:
Yep, but if rather do that when we come to render a preview. This was just something quick and dirty to keep moving.



Regards,


Andrew Garner

Guidance Guarantee Programme
HM Treasury

+44 7971 232 140

On Thu, Mar 19, 2015 at 5:22 PM, Gareth Visagie <notifications@github.com>
wrote:

>> @@ -162,7 +162,16 @@
>>  
>>      <p><b>Total value of your pension pot:</b></p>
>>  
>> -    <p class="data"><%= @appointment_summary.value_of_pension_pots %></p>
>> +    <p class="data">
>> +      <% if @appointment_summary.value_of_pension_pots.blank? %>
>> +        Unknown
>> +      <% elsif @appointment_summary.upper_value_of_pension_pots.blank? %>
>> +        <%= number_to_currency @appointment_summary.value_of_pension_pots, precision: 0 %>
>> +      <% else %>
>> +        <%= number_to_currency(@appointment_summary.value_of_pension_pots, precision: 0) + ' - ' +
>> +                number_to_currency(@appointment_summary.upper_value_of_pension_pots, precision: 0) %>
>> +      <% end %>
> Can we move all of this logic to a presenter?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/guidance-guarantee-programme/output/pull/54/files#r26776595